src/main/java/uk/gov/hmcts/reform/em/hrs/service/SegmentDownloadServiceImpl.java

```diff
@@ -43,41 +41,32 @@ public SegmentDownloadServiceImpl(HearingRecordingSegmentRepository segmentRepos
         this.auditEntryService = auditEntryService;
     }
 
+
     @Override
-    @PreAuthorize("hasPermission(#recordingId,'READ')")
-    public Map<String, String> getDownloadInfo(UUID recordingId, Integer segmentNo) {
+    public HearingRecordingSegment fetchSegmentByRecordingIdAndSegmentNumber(UUID recordingId, Integer segmentNo) {
 
         HearingRecordingSegment segment =
             segmentRepository.findByHearingRecordingIdAndRecordingSegment(recordingId, segmentNo);
-
-        auditEntryService.createAndSaveEntry(segment, AuditActions.USER_DOWNLOAD_REQUESTED);
-
-        BlobProperties blobProperties = blobstoreClient.getBlobProperties(segment.getFilename());
-
-        LOGGER.info(
-            "downloading blob with the following properties: [filenmae: {}, content-type: {}, content-length: {}]",
-            segment.getFilename(), blobProperties.getContentType(), blobProperties.getBlobSize()
-        );
-
-        return Map.of(
-            "filename", segment.getFilename(),
-            "contentType", blobProperties.getContentType(),
-            "contentLength", String.valueOf(blobProperties.getBlobSize())
```

Comment:
do we not need to set these details anymore?

Reply:
have moved - please see comments below as to where